src/components/templates/PosterPage/components/ShareModal/ShareModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC, useState, useCallback } from "react";
+import { Modal } from "react-bootstrap";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { faFacebook, faTwitter } from "@fortawesome/free-brands-svg-icons";
+import { DEFAULT_MAP_BACKGROUND } from "settings";
+import { WithId } from "utils/id";
+import { PosterPageVenue } from "types/venues";
+import classNames from "classnames";
+import { LinkButton } from "components/atoms/LinkButton";
+
+import "./ShareModal.scss";
+
+interface ConfirmationModalProps {
+  show: boolean;
+  onHide: () => void;
+  venue: WithId<PosterPageVenue>;
+}
+
+export const ShareModal: FC<ConfirmationModalProps> = ({
+  show,
+  onHide,
+  venue,
+}) => {
+  const url = `${process.env.REACT_APP_CODE_CHECK_URL}in/${venue.id}`;
```

Comment:
should be moved to utils? @0xdevalias wdyt, where and how to name it. Also, I'm not sure this is the best way to generate the link to the current place, but I don't have the capacity to dive into the options

Reply:
Why is `REACT_APP_CODE_CHECK_URL` getting used here? Pretty sure that relates to a legacy/no longer used external microservice.

What's the high level aim/intent here? To get the full URL of the current page/venue/similar? If so, look at `utils/url`. There may not be an exact function there currently, but there should be the basic pattern to use, eg. `new URL(`/in/${venue.id}`, window.location.origin)`

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/99a584cdd3729c9db9813f7e7b941d9c6bcba1e4